pkgs/applications/editors/uemacs/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchgit, ncurses }:
+
+stdenv.mkDerivation rec {
+  name = "uemacs-${version}";
+  version = "2014-12-08";
+
+  src = fetchgit {
+    url = git://git.kernel.org/pub/scm/editors/uemacs/uemacs.git;
+    rev = "8841922689769960fa074fbb053cb8507f2f3ed9";
+    sha256 = "14yq7kpkax111cg6k7i3mnqk7sq7a65krq6qizzj7vvnm7bsj3sd";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace "-lcurses" "-lncurses" \
+      --replace "CFLAGS=-O2" "CFLAGS+=" \
+      --replace "BINDIR=/usr/bin" "BINDIR=$out/bin" \
+      --replace "LIBDIR=/usr/lib" "LIBDIR=$out/usr/share/uemacs"
+    substituteInPlace epath.h \
+      --replace "/usr/global/lib/" "$out/usr/share/uemacs/" \
```

Comment:
yeah I thought the same but saw it being used in another package so I put it in.
I'll fix it up as soon as I'm done with a big package I'm currently compiling.